docker-bake.hcl

```diff
@@ -15,3 +15,27 @@ target "azure" {
     platforms = ["linux/amd64", "linux/arm64"]
     args = {"BASE_IMAGE": "alpine:3.21"}
 }
+
+target "aws-fips" {
+    target = "aws"
+    platforms = ["linux/amd64", "linux/arm64"]
+    args = {
+        "BASE_IMAGE": "ghcr.io/spacelift-io/apline-fips:base-latest"
```

Comment:
`apline`

Reply:
yeah, Im a dumb dumb I caught it 😂 just pushed the fix.